### PR TITLE
Jumlfile

### LINKED ISSUE #220: Improve the passing of args
So that passing `--option-key value1 value2` is possible.

### PR DESCRIPTION
**Pull request by Dufgui**
Fixes #220  .

Changes proposed in this Pull Request:
  - load jumlfile
  - create load option unit test

### CODE REVIEW

**Comment by Dufgui** [comment]
if it's ok and when it' integrated i will update the doc

File: lib/jhipsteruml.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ const options = {
   angularSuffixes: (argv['angular-suffix']) ? jhipsterOptionHelper.askForAngularSuffixes(parsedData.classes, argv['angular-suffix']) : {},
   microserviceNames: (argv['microservice-name']) ? jhipsterOptionHelper.askForMicroserviceNames(parsedData.classes, argv['microservice-name']) : {},
   searchEngines: (argv['search-engine']) ? jhipsterOptionHelper.askForSearchEngines(parsedData.classes, argv['search-engine']) : {},
-  noFluentMethods: (argv['no-fluent-methods']) ? jhipsterOptionHelper.askForClassesWithNoFluentMethods(parsedData.classes) : []
+  noFluentMethods: (! argv['fluent-methods']) ? jhipsterOptionHelper.askForClassesWithNoFluentMethods(parsedData.classes) : []
```
Comment (MathieuAA) [inline lib/jhipsteruml.js]: Maybe we could just change `noFluentMethods` to `fluentMethods`, don't you think?

Comment (Dufgui) [inline lib/jhipsteruml.js]: Ok

**Comment by Dufgui** [comment]
No but I can. I test it too

**Comment by MathieuAA** [comment]
Thanks, from what I've seen, it could be tricky

**Comment by Dufgui** [comment]
What bug / problem have you seen  with the 7.X version ?
I don't see anything with my unit tests. Neither in changelog : https://github.com/yargs/yargs/blob/master/CHANGELOG.md

**Comment by MathieuAA** [comment]
I was mistaken, I actually wrote something code in my test...

**Comment by Dufgui** [comment]
I see something strange. We call well the entity generator with or without the flag no-fluent-methods.
But what ever we choose the json entity file contain "fluentMethods : true".
It came from https://github.com/jhipster/jhipster-uml/blob/master/lib/entitiescreator.js#L82
wich never change.
I think it's a bug no? I create an issue?

**Comment by MathieuAA** [comment]
It's not, as it's the default setting for the option. The problem is that it's actually never set to what the user wants.

**Comment by Dufgui** [comment]
So I finish this and I think the rest must be in another issue no ?

**Comment by MathieuAA** [comment]
Yes, if I finish my improvement for the JDL today, I'll fix it myself.

**Comment by Dufgui** [comment]
I let you merge it :)

**Comment by MathieuAA** [comment]
Thanks =)

**Comment by Dufgui** [comment]
your welcome

**Pull request merged**